### PR TITLE
feat(NewOrderAhead/Order Confirmation): Added the order confirmation screen layout. Also added the Back Button which now will be integrated with other pages too.

### PR DESCRIPTION
**Pull request by dabralyogesh12**
Issues: fixes https://github.com/SleekOrg/tracker/issues/296
feat(NewOrderAhead/Order Confirmation): Added the order confirmation screen layout. Also added the Back Button which now will be integrated with other pages too.
Description: Added the layout for the Order confirmation screen. The font, for now, is picked from the central theme file so can be GT Walsheim somewhere instead of Roboto.
Verification: Tested locally.

### CODE REVIEW

File: src/features/style/style.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.PhoneInputInput {
```
Comment (teamSleek) [inline src/features/style/style.css:1]: Can we create a sleek-phone-input-box using react-phone-input-box and use that? we should also scope all following css changes to `sleek-phone-input-box` only.

File: src/features/stall/OrderConfirmation.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  Paper,
+} from '@material-ui/core';
+import React from 'react';
+import PhoneInput from 'react-phone-number-input';
+import 'react-phone-number-input/style.css';
+import OrderImage from '../../assets/OrderConfirm.png';
+import { BackButton } from '../../components/BackButton';
+import '../style/style.css';
+
+const useStyles = makeStyles((theme: Theme) =>
```
Comment (teamSleek) [inline src/features/stall/OrderConfirmation.tsx:20]: The font seems off. Let use the correct font as we build new things as the theme file is now settled.

**Review by teamSleek — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 3]
```diff
@@ -64,6 +64,11 @@ class App extends Component<IProps, IState> {
               <Route exact path="/stall/menu" component={StallMenu} />
               <Route exact path="/stall/cart/added" component={AddedToCart} />
               <Route exact path="/stall/order/sent" component={OrderSent} />
+              <Route
```
[the remark this answers is not in the document]
Comment (dabralyogesh12) [reply src/App.tsx:67]: Done, updated PR.

File: src/features/style/style.css (hunk 1 above)
Comment (dabralyogesh12) [reply src/features/style/style.css]: Done, updated PR.

File: src/features/stall/OrderConfirmation.tsx (hunk 2 above)
Comment (dabralyogesh12) [reply src/features/stall/OrderConfirmation.tsx:20]: Done, updated PR.

File: src/App.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (dabralyogesh12) [reply src/App.tsx:67]: Done, updated PR.

File: package.json [hunk 4]
```diff
@@ -16,6 +16,7 @@
     "@types/node": "^12.20.6",
     "@types/react": "^17.0.3",
     "@types/react-dom": "^17.0.3",
+    "@types/react-phone-number-input": "^3.0.6",
```
Comment (teamSleek) [inline package.json:19]: it seems that package-lock.json is broken? I see no deletion from package.json and but so many lines deleted from package-lock.json

File: src/features/stall/OrderConfirmation.tsx [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+      marginTop: theme.spacing(1.2),
+      marginBottom: theme.spacing(2.4),
+    },
+    notificationText: {
+      textTransform: 'none',
+      color: 'white',
+      textAlign: 'center',
+    },
+    notificationPaper: {
+      background: '#6D5CFF',
```
Comment (teamSleek) [inline src/features/stall/OrderConfirmation.tsx:65]: same concern about color referencing as other pr

**Review by teamSleek — CHANGES_REQUESTED** [review]

File: src/features/stall/OrderConfirmation.tsx (hunk 5 above)
Comment (dabralyogesh12) [reply src/features/stall/OrderConfirmation.tsx]: Done, updated PR.

**Pull request merged**